### PR TITLE
translate cryptographic hash function for zh-CN

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/glossary/cryptographic_hash_function/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: 加密散列函数
+slug: Glossary/Cryptographic_hash_function
+---
+
+{{GlossarySidebar}}
+
+加密散列函数有时也称为*摘要函数*，其是一种{{glossary("cryptography", "加密")}}原语，将任意大小的信息转位固定大小的消息，称为{{glossary("digest", "摘要")}}。加密散列函数通常用于授权、{{Glossary("digital signature", "数字签名")}}以及{{Glossary("HMAC", "密钥散列消息认证码")}}.
```
Comment (fwqaaq) [inline files/zh-cn/glossary/cryptographic_hash_function/index.md:8]:
```suggestion
加密散列函数有时也称为*摘要函数*，其是一种{{glossary("cryptography", "加密")}}原语，将任意大小的信息转位固定大小的消息，称为{{glossary("digest", "摘要")}}。加密散列函数通常用于授权、{{Glossary("digital signature", "数字签名")}}以及{{Glossary("HMAC", "密钥散列消息认证码")}}。
```

File: files/zh-cn/glossary/cryptographic_hash_function/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: 加密散列函数
```
Comment (yin1999) [inline files/zh-cn/glossary/cryptographic_hash_function/index.md:2]: actually, I don't like the translation, as hash is not a method for encryption. I'd prefer "密码学" or "密码", just like the first translation given by wikipedia.

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/glossary/cryptographic_hash_function/index.md (hunk 2 above)
Comment (fwqaaq) [reply files/zh-cn/glossary/cryptographic_hash_function/index.md:2]: Yeah, you're right, I'll modify not so good title.

```suggestion
title: 密码散列函数
```

File: files/zh-cn/glossary/cryptographic_hash_function/index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: 密码散列函数
+slug: Glossary/Cryptographic_hash_function
+---
+
+{{GlossarySidebar}}
+
+加密散列函数有时也称为*摘要函数*，其是一种{{glossary("cryptography", "密码")}}原语，将任意大小的信息转为固定大小的消息，称为{{glossary("digest", "摘要")}}。加密散列函数通常用于授权、{{Glossary("digital signature", "数字签名")}}以及{{Glossary("HMAC", "密钥散列消息认证码")}}。
+
+散列函数必须有以下特征才能用于加密：
```
Comment (yin1999) [inline files/zh-cn/glossary/cryptographic_hash_function/index.md:10]:
```suggestion
散列函数必须有以下特征才能用于密码学：
```

File: files/zh-cn/glossary/cryptographic_hash_function/index.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- 快速计算（因为它们需要频繁的生成）
```
Comment (yin1999) [inline files/zh-cn/glossary/cryptographic_hash_function/index.md:12]:
```suggestion
- 快速计算（因为它们需要频繁地生成）
```

File: files/zh-cn/glossary/cryptographic_hash_function/index.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+- 不可逆（每个摘要可能来自拥有庞大数据的消息，并且仅有暴力破解才能生成导致给定摘要的消息）
+- 防篡改（对消息的任何改变都会导致不同的摘要）
```
Comment (yin1999) [inline files/zh-cn/glossary/cryptographic_hash_function/index.md:14]:
```suggestion
- 不可逆（每个摘要可能来自拥有庞大数据的消息，并且仅有暴力破解才能生成产生给定摘要的消息）
- 防篡改（对消息的任何改变都会产生不同的摘要）
```

File: files/zh-cn/glossary/cryptographic_hash_function/index.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+- 抗碰撞（应该不可能找到两个不同的消息产生相同的摘要）
+
+像 MD5 和 SHA-1 这样的加密散列函数被认为是不安全的，因为已经发现了一种可以显著降低它们抗碰撞性的一种方式。
```
Comment (yin1999) [inline files/zh-cn/glossary/cryptographic_hash_function/index.md:17]:
```suggestion
像 MD5 和 SHA-1 这样的加密散列函数被认为是不安全的，因为已经发现了可以显著降低它们抗碰撞性的攻击。
```

**Review by yin1999 — APPROVED** [review]
Thx

**Pull request merged**